lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+  attr_reader :MENU
+
+  MENU = { "hotdog"=>1.99,
+          "cheesebuger"=>1.49,
+          "Hamburger"=>0.99,
+          "Fries"=>0.89, }
+
+    def initialize(menu = MENU)
+      @menu = menu
+    end
+
+    def read_menu
+      MENU
```

Comment:
should be `@menu` instead of `MENU`. However this entire method is unnecessary as you have an attr_reader.